Description: Compile also WebStartAlphabetRegistration
 For some reason I'm now aware of, the class
 WebStartAlphabetRegistration doesn't get compiled with standard
 build.xml. This patch fixes this behavior.
Author: Giovanni Mascellani <gio@debian.org>
Forwarded: no
Last-Update: 2011-12-08

--- libjlatexmath-java-0.9.6.orig/build.xml
+++ libjlatexmath-java-0.9.6/build.xml
@@ -74,7 +74,7 @@
 
   <!-- Compile source files -->
   <target name="compile" depends="init">
-    <javac includeantruntime="false" srcdir="${project.src.dir}" destdir="${project.bin.dir}" debug="on" source="${java_version}" target="${java_version}" excludes="**/WebStartAlphabetRegistration.java">
+    <javac includeantruntime="false" srcdir="${project.src.dir}" destdir="${project.bin.dir}" debug="on" source="${java_version}" target="${java_version}">
       <!--compilerarg value="-Xlint"/-->
     </javac>
   </target>
